internal/lm/nvml.go

```diff
@@ -218,6 +232,41 @@ func newGPUModeLabeler(devices []resource.Device) (Labeler, error) {
 	return labels, nil
 }
 
+func newImexDomainLabeler(configFile string) (Labeler, error) {
+	// read file and parse it
+	imexConfig, err := os.Open(configFile)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read imex config file: %v", err)
+	}
+	defer imexConfig.Close()
```

Comment:
This shouldn't error out -- it should simply not apply the label if the file is not there (or is empty).

Reply:
done, now logs a warning and returns nil

Reply:
I'm not sure we should even log a warning -- I'm assuming most systems wont have this file.